src/sentry/migrations/0800_rm_incidentseen_incidentsubscription.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 5.1.1 on 2024-12-03 18:48
+import django
+from django.db import migrations
+
+import sentry
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.new_migrations.monkey.models import SafeDeleteModel
+from sentry.new_migrations.monkey.state import DeletionAction
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0799_cron_incident_index"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="incidentseen",
+            name="incident",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                db_constraint=False,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="sentry.incident",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="incidentseen",
+            name="user_id",
+            field=sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                "sentry.User", db_index=False, null=True, on_delete="CASCADE"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="incidentsubscription",
+            name="incident",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                db_constraint=False,
+                db_index=False,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="sentry.incident",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="incidentsubscription",
+            name="user_id",
+            field=sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                "sentry.User", db_index=True, null=True, on_delete="CASCADE"
+            ),
+        ),
```

Comment:
When I generated the 2nd part of removing the models from code it also generated this:

```
migrations.AlterUniqueTogether(
    name="incidentsubscription",
    unique_together=None,
),
SafeRemoveField(
    model_name="incidentsubscription",
    name="incident",
    deletion_action=DeletionAction.MOVE_TO_PENDING,
),
```

but it did the same in https://github.com/getsentry/sentry/pull/81264 and I had removed it without issue.

Reply:
Since you're going to be dropping the table shortly, the `AlterUniqueTogether` operation isn't that important as postgres will drop the constraint when the table is dropped.